lib/tasks/seed_users_with_profiles.rake

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+USER_COUNT = 100
+USER_PASSWORD = 'salty pickles'
+
+namespace :seed do
+  desc 'Generate users with broken profile data for local development'
+  task broken_profiles: :environment do
+    warn 'Starting seeder...'
+    warn "Creating data for #{USER_COUNT} users..."
+    (1..USER_COUNT).to_a.each do |num|
+      create_user
+      warn "#{num}/#{USER_COUNT}"
+    end
+    warn 'Complete!'
+  end
+
+  def create_user
+    email = "#{Faker::Name.first_name.downcase}_#{Faker::Name.last_name.downcase}@example.com"
+    if User.find_with_email(email)
+      warn 'User skipped: Duplicate email'
+      return
+    end
+    user = User.create!(
+      confirmed_at: Time.zone.now,
+      email: email,
+      password: USER_PASSWORD,
+      personal_key: RandomPhrase.new(num_words: 4).to_s,
+    )
+    user.phone_configurations.create!(
+      phone_configuration_data(user, num: 1234),
+    )
+    user.email_addresses.create!(
+      confirmed_at: Time.zone.now,
+      email: email,
+    )
+    enrollment = create_cancelled_enrollment(user)
+    create_profile(user, enrollment)
+    warn "Created user with email= #{email}"
+  end
+
+  def phone_configuration_data(user, args)
+    {
+      delivery_preference: user.otp_delivery_preference,
+      phone: format('+1 (415) 555-%04d', args[:num]),
+      confirmed_at: Time.zone.now,
+    }
+  end
+
+  def create_cancelled_enrollment(user)
+    enrollment = InPersonEnrollment.find_or_initialize_by(
+      user: user,
+      status: [:expired, :cancelled, :failed].sample,
+      sponsor_id: IdentityConfig.store.usps_ipp_sponsor_id,
+    )
+    enrollment.save!
+    enrollment
+  end
+
+  def create_profile(user, enrollment)
+    profile = Profile.new(
+      user: user,
+      active: false,
+      in_person_verification_pending_at: enrollment.created_at,
+    )
+    pii = Pii::Attributes.new_from_hash(
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      dob: Faker::Date.between(from: '1940-01-01', to: '2006-12-31'),
+      ssn: "666-#{Faker::Number.number(digits: 2)}-#{Faker::Number.number(digits: 3)}",
+      address1: Faker::Address.street_address,
+      city: Faker::Address.city,
+      state: Faker::Address.state_abbr,
+      zipcode: Faker::Address.zip_code,
+      identity_doc_address1: Faker::Address.street_address,
+      identity_doc_city: Faker::Address.city,
+      identity_doc_address_state: Faker::Address.state_abbr,
+      identity_doc_zipcode: Faker::Address.zip_code,
+    )
+    profile.encrypt_pii(pii, USER_PASSWORD)
+    profile.save!
+
+    enrollment.profile = profile
+    enrollment.save!
+  end
+end
```

Comment:
For `Run the backfill script to backup data without updating profiles` - we are just running that to 1. confirm batches are working and 2. ensure the print out is working as expected, correct? What do you think about adding a sentence on the testing plan to confirm both? (I am adding it here so it can be resolved.)